client/cli/src/main.rs

```diff
@@ -204,5 +207,18 @@ async fn main() {
         Command::WaitBlockHeight { height } => {
             client.await_consensus_block_height(height).await;
         }
+        Command::Gateways {} => {
+            // List any gateways registered with the federation.
+            println!("Fetching gateways from federation...");
+            let gateways = client
+                .select_gateways()
+                .await
+                .expect("Failed to fetch gateways");
+            println!("Found {} registered gateways : ", gateways.len());
+
+            for (_, gateway) in gateways.iter().enumerate() {
+                println!("{}", serde_json::to_string(&gateway).unwrap());
+            }
```

Comment:
If we are going with JSON, why not just pretty-print the whole vec of gateways as JSON?

Reply:
Let's just print out the vec returned by `client.fetch_gateways()`.

Reply:
But maybe add an `active: true` key to any gateway that matches the one saved in `LightningGatewayKey` in the client database. Also, probably want to put that one at the beginning of the list.